free/src/main/scala/cats/free/Coyoneda.scala

```diff
@@ -17,29 +18,32 @@ sealed abstract class Coyoneda[F[_], A] extends Serializable { self =>
   /** The underlying value. */
   val fi: F[Pivot]
 
-  /** The transformer function, to be lifted into `F` by `run`. */
-  val k: Pivot => A
+  /** The transformer functions, to be lifted into `F` by `run`. */
+  private[cats] val k: List[Any => Any]
```

Comment:
what if we called this `ks` and keep `k: Pivot => A = execute _` so we don't break binary compatibility here?